tests/sandbox_test.rs

```diff
@@ -23,23 +44,23 @@ mod tests {
         assert_eq!(123, account_info.amount);
     }
 
-    /*
     #[test]
-    fn serum_v1_setup() {
+    fn order_crank_market() {
         let sandbox = Sandbox::new().unwrap();
-        let actor = Actor::new(&sandbox);
-        actor.airdrop(10 * LAMPORTS_PER_SOL).unwrap();
-        let base_mint = Mint::new(&sandbox, &actor, 0, None, None).unwrap();
-        let quote_mint = Mint::new(&sandbox, &actor, 0, None, None).unwrap();
-        let serum_program = actor
+        println!("{}", sandbox.url());
+        let market_creator = Actor::new(&sandbox);
+        market_creator.airdrop(10 * LAMPORTS_PER_SOL).unwrap();
+        let base_mint = Mint::new(&sandbox, &market_creator, 0, None, None).unwrap();
+        let quote_mint = Mint::new(&sandbox, &market_creator, 0, None, None).unwrap();
+        let serum_program = market_creator
             .deploy(&std::path::Path::new(
-                "/home/yfang/serum-dex/dex/target/deploy/serum_dex.so",
+                "/home/rprasadintern/work/serum-dex/dex/target/deploy/serum_dex.so",
```

Comment:
Can we kill this hard-coded .so in favor of downloading an artifact or building locally?